schemas/event.yml

```diff
@@ -277,6 +277,14 @@
             - end
             - info
             - start
+        - name: registry
+          description: >
+            Use this category of event for events related to the Windows registry.
+          expected_event_types:
```

Comment:
thanks for the add.
I think this seems like a good start. And I would err on the side of omitting `info` like you did. It's always easier to add later than it is to remove.

These values may not be specific enough, because they don't differentiate between key vs value changes. But I think 99% of the time, we're looking at changed values, not keys. So this is okay in my opinion.

Reply:
@MikePaquette Did you have a specific use in mind for "info"? This proposal is not yet adding it as an expected value.

One way Auditbeat uses "info" is to dump a list of all running processes, to establish a baseline. Then it reports process start/end as things happen.

I could see a similar use for dumping a set of registry keys (hopefully not the whole hive); then monitor for changes with creation/change/deletion. Was that the idea?

Reply:
@rw-access On specificity wrt key vs value activity, I'm not sure we should change the allowed values in `event.type`.

I would think `event.action` can be used to capture whether it's a change to a value or a data blob?